### PR TITLE
urns #1: Add schemes field to allow channels to support more than one scheme

### PR DESCRIPTION
**Pull request by nicpottier**
This keeps the `scheme` field for now, but adds a new `schemes` field which it backfills. That let us ship similar changes to mage without breaking anything. (I'll roll those into the TwitterID scheme changes when we do that)

### CODE REVIEW

File: temba/channels/migrations/0074_channel_schemes.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+def populate_schemes(apps, schema_editor):
+    Channel = apps.get_model('channels', 'Channel')
+
+    # find all channels which aren't 'tel', update their schemes appropriately
```
Comment (rowanseymour) [inline temba/channels/migrations/0074_channel_schemes.py]: Why not tel?

Comment (nicpottier) [inline temba/channels/migrations/0074_channel_schemes.py]: Because tel is the default value for that field, so already set in the migration.

**Comment by nicpottier** [comment]
Is that true? It is the same transaction no? So should all be done at once..

**Comment by rowanseymour** [comment]
Not sure now... what happens if another server tries to add a new channel before that transaction completes?

**Comment by nicpottier** [comment]
It'll block until this is done.

**Comment by nicpottier** [comment]
So I guess the possibility is getting one that has a default `['tel']` schemes that shouldn't.

**Pull request merged**